server/controllers/finance/fiscal.js

```diff
@@ -618,7 +618,10 @@ function closing(req, res, next) {
     })
     .then((rows) => {
       const queryUpdateFiscal = rows.pop();
-      if (!queryUpdateFiscal.changedRows) {
+
+      // Note: changedRows is generated from the server message.
+      // It is not correctly parsed and set in multi-langual environments.
+      if (!queryUpdateFiscal.affectedRows) {
```

Comment:
In the future, changes like this should be on a separate branch.  This is a bug fix that needs to get in as soon as possible - it has nothing to do with employees.